src/Builder/ClassModifier/ClassInstanceModifierStrategy/Unwire/UnwireReturnArrayModifierStrategy.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Integrator\Builder\ClassModifier\ClassInstanceModifierStrategy\Unwire;
+
+use PhpParser\Node\Stmt\ClassMethod;
+use SprykerSdk\Integrator\Builder\ClassModifier\AddVisitorsTrait;
+use SprykerSdk\Integrator\Builder\ClassModifier\ClassInstanceModifierStrategy\Applicable\ApplicableInterface;
+use SprykerSdk\Integrator\Builder\Visitor\RemovePluginFromPluginListVisitor;
+use SprykerSdk\Integrator\Transfer\ClassInformationTransfer;
+use SprykerSdk\Integrator\Transfer\ClassMetadataTransfer;
+
+class UnwireReturnArrayModifierStrategy implements UnwireModifierStrategyInterface
```

Comment:
Let's follow the rule for class naming`{className}{implementedInterface}`, e.g if an interface name `UnwireModifierStrategyInterface` the class name should be `ReturnArrayUnwireModifierStrategy`.
Could you please to check all introduced class names?

Reply:
Renamed by pattern `{className}{implementedInterface}`